tests/test_docker_latex.py

```diff
@@ -0,0 +1,51 @@
+import os
+import shutil
+import unittest
+import logging
+
+import quizgen.latex
+import quizgen.pdf
+import quizgen.util.dirent
+import quizgen.util.json
+import tests.base
+
+class TestDockerPdfConversion(tests.base.BaseTest):
+    def test_quiz_to_pdf(self):
+        if not quizgen.latex.is_available(use_docker = True):
+            self.skipTest("Docker is not installed or not running")
+
+        source_quiz_path = tests.base.discover_good_quiz_files()[0]
+
+        if not os.path.exists(source_quiz_path):
+            self.skipTest(f"Quiz file '{source_quiz_path}' not found")
```

Comment:
This is bananas.
You just discovered the quiz, but then just skip a test if it doesn't exist?
As already commented above, I don't think you should discover here.
But if you were, this sort of behavior shouldn't warrant a skip, it should be a failure.
This may seem like a small thing, but it is very important that you keep in mind what the "right thing" to do it and always do it.
We don't like shortcuts in this project.